packages/hops/lib/runtime.d.ts

```diff
@@ -0,0 +1,83 @@
+import { BrowserRouterProps } from 'react-router-dom';
+
+declare module 'hops' {
+  type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+  export class Miss extends React.Component<any, any> {}
```

Comment:
Still a typescript newbie but shouldn't it be `React.Component<{}, any>` - no props allowed?

Reply:
I've removed the `any`s now, so that we are using the default values, which are `{}`.